recipes/synphot/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "synphot" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c512ecd3dd2fb46830c8e99036f14e5bf03e8b624bd3c931e46676ff0a78b304
+  patches:
+    - setup.cfg.patch
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
```

The command can be simplified as the options are now part of the default.

```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```